pymeasure/instruments/anaheimautomation/dpseriesmotorcontroller.py

```diff
@@ -244,12 +244,12 @@ def step_position(self, pos):
     @property
     def absolute_position(self):
         """ Float property representing the value of the motor position measured in absolute units.
-        Note that in DP series motor controller instrument manuals, `absolute position` refers to
-        the 'step_position' property rather than this property. Also note that use of this property
-        relies on steps_to_absolute() and absolute_to_steps() being implemented in a subclass. In
-        this way, the user can define the conversion from a motor step position into any desired
-        absolute unit. Absolute units could be the position in meters of a linear stage or the
-        angular position of a gimbal mount, etc. This property can be set.
+        Note that in DP series motor controller instrument manuals, 'absolute position' refers to
+        the ``step_position`` property rather than this property. Also note that use of this
```

Comment:
```suggestion
        the :attr:`step_position` property rather than this property. Also note that use of this
```
That should work as a link to that property. Please check that it works as expected.

Reply:
Updated in last commit